app/javascript/react/screens/App/Mappings/Mappings.js

```diff
@@ -166,9 +165,7 @@ class Mappings extends Component {
       <React.Fragment>
         <Toolbar>
           <Breadcrumb.Item href="/dashboard/maintab?tab=compute">{__('Compute')}</Breadcrumb.Item>
-          <li>
-            <Link to="/">{__('Migration')}</Link>
-          </li>
+          <Breadcrumb.Item href="/migration">{__('Migration')}</Breadcrumb.Item>
```

Comment:
Pity, we can't take advantage of the more efficient `react-router-dom`

I'm facing a similar issue in https://github.com/ManageIQ/manageiq-v2v/pull/661 ...
When in the Migration screen, the `View Details` link does a full page load when transitioning to the Plan Detail page